samples/v1beta1/conftest.py

```diff
@@ -76,7 +76,21 @@ def random_entry_name(client, entry_group_name):
     )
     random_entry_name = "{}/entries/{}".format(entry_group_name, random_entry_id)
     yield random_entry_name
-    client.delete_entry(request = {'name': random_entry_name})
+    client.delete_entry(request={"name": random_entry_name})
+
+
+@pytest.fixture
+def entry(client, entry_group_name):
+    now = datetime.datetime.now()
+    random_entry_id = "example_entry_{}_{}".format(
+        now.strftime("%Y%m%d%H%M%S"), uuid.uuid4().hex[:8]
+    )
+    entry = datacatalog_v1beta1.CreateEntryRequest
+    entry = client.create_entry(
+        request={"parent": entry_group_name, "entry_id": random_entry_id, "entry": {"type": "DATA_STREAM", "name": "samples_test_entry"}}
```

Comment:
Should we use `type_` here?

The proto files seems to have [`type_`](https://github.com/googleapis/python-datacatalog/blob/b2c8892d9e900d8b353ef6f035fc7c0f4f75a974/google/cloud/datacatalog_v1beta1/types/datacatalog.py#L493).

Reply:
No, `type_` doesn't work.

Reply:
Nevermind, you're 100% right. Was pinned to the wrong version locally from requirements.txt. Updated.